Disclaimer : i use arcgis library to get the data

In [1]:
import pandas as pd
from arcgis.features import FeatureLayer
import csv

In [2]:
url = 'http://gis.dukcapil.kemendagri.go.id/arcgis/rest/services/2017semester1/MapServer/5'

In [3]:
layer= FeatureLayer(url)

In [4]:
df = pd.DataFrame(layer.query().df)
df.shape

(7187, 212)

In [5]:
df.head()

a  a_  a_2  ab  ab_  ab_2  agama_dan_  aparatur_p    b  b_  \
objectid                                                                 
1          80   0    0  23    0     1           9         226   99   0   
2         304   1    2  88    0     4          29         707  323   4   
3         117   1    1  51    0     0          17         366  137   2   
4         273   0    1  81    1     2          19         552  311   1   
5         268   0    0  72    0     2          26         529  290   1   

             ...        u5   u50  u55  u60  u65  u70  u75  wajib_ktp  wanita  \
objectid     ...                                                               
1            ...       477   281  208  147   76   47   60       3567    2603   
2            ...      2046  1204  996  687  479  281  507      16634   11560   
3            ...      1084   779  605  416  313  202  298      10177    6979   
4            ...      1087   755  641  464  312  253  308       9863    6752   
5            ...      1716  1182  944  622  449  264  568      15433   10568   

          wiraswasta  
objectid              
1                530  
2               2236  
3               1472  
4               2025  
5               2259  

[5 rows x 212 columns]

This data have 212 features but not clean actually, so many noise, so i cut the features just i need.

jumlah_pen_ is sum of population, kepadatan_ is density of population, nama_.. is description of district, province and soon..

In [6]:
df_ = df[['jumlah_kk', 'jumlah_pen', 'kepadatan_','nama_kab_s', 'nama_kec_s', 'nama_prop_']]

In [7]:
df_.to_csv('jml_pend_kec.csv', index=False)